var/spack/repos/builtin/packages/libpng/package.py

```diff
@@ -34,6 +38,8 @@ def configure_args(self):
             f"CPPFLAGS={self.spec['zlib'].headers.include_flags}",
             f"LDFLAGS={self.spec['zlib'].libs.search_flags}",
         ]
+        if "~shared" in self.spec:
+            args += ["--enable-static", "--disable-shared"]
```

Comment:
If both shared and static can be built at the same time, can we add a `libs` variant like e.g. in the `lzo` package:
```python
 variant(
        "libs",
        default="shared,static",
        values=("shared", "static"),
        multi=True,
        description="Build shared libs, static libs or both",
    )
```
?

Reply:
Yeah I'm good with that, I'll add it and test it out.